modules/olm-installing-from-operatorhub-using-cli.adoc

```diff
@@ -210,15 +316,33 @@ spec:
 <3> Include the subscription ID.
 --
 
-. Create the `Subscription` object:
+.. Create the `Subscription` object:
 +
 [source,terminal]
 ----
-$ oc apply -f sub.yaml
+$ oc apply -f subscription.yaml
 ----
-+
+
+. If you set the `installPlanApproval` field to `Manual`, manually approve the pending install plan to complete the Operator installation.
```

Comment:
>  manually approve the pending install plan

I assume it will be obvious how to manually approve?

Reply:
It was linked in "Additional resources", but I've added the following sentence to make that connection more clear:
> For more information, see "Manually approving a pending Operator update".